pages/index.tsx

```diff
@@ -1,74 +1,68 @@
-import type { NextPage } from 'next';
-import Head from 'next/head';
-import Image from 'next/image';
-import styles from '../styles/Home.module.scss';
+import type { NextPage } from 'next'
+import Head from 'next/head'
+import Image from 'next/image'
+import styles from '../styles/Home.module.scss'
 
 const Home: NextPage = () => (
-  <div className={styles.container}>
-    <Head>
-      <title>Create Next App</title>
-      <meta name="description" content="Generated by create next app" />
-      <link rel="icon" href="/favicon.ico" />
-    </Head>
+	<div className={styles.container}>
+		<Head>
+			<title>Create Next App</title>
+			<meta name='description' content='Generated by create next app' />
+			<link rel='icon' href='/favicon.ico' />
+		</Head>
```

Comment:
Indentation of 2 spaces. This does not look good.

Reply:
This happened on its own when I committed the files. I will take a look once in this issue.

Reply:
Might be some prettierconfig error.
Try changing it.